### ISSUE TITLE
Where do the dispatch and action calls come from? 

### ISSUE THREAD

**Issue by mrdulin** [clarkbw/jest-localstorage-mock#124]
There is no mention of redux in the documentation for the [test examples](https://github.com/clarkbw/jest-localstorage-mock#test-examples)

**Comment by clarkbw** [comment]
Yes, that's a redux example.  Should probably note that above the example.

Would you mind creating a PR to update the README?

**Comment by mrdulin** [comment]
@clarkbw  Is it necessary to introduce redux examples? I think only the basic usage needs to be given.

**Comment by clarkbw** [comment]
I think it helps show the async nature of testing.

**Comment by kendallroth** [comment]
While I do agree that these examples are beneficial, I would suggest that most of the examples should simply use `localStorage.setItem` and `localStorage.getItem` calls (as they are not directly documented anywhere) to reduce confusion.

**Comment by Chiron1991** [comment]
+1 to remove them from the readme. I just checked `jest-localstorage-mock` out for the first time in a project that doesn't even use redux and was slightly confused until I discovered this issue.
@clarkbw Would you agree to remove them if I file a PR?
